includes/Dashboard/Templates/Settings.php

```diff
@@ -186,18 +188,102 @@ public function load_store_content() {
      *
      * @since 2.4
      *
+     * @param string $slug_suffix
+     *
      * @return void
      */
-    public function load_payment_content() {
+    public function load_payment_content( $slug_suffix ) {
         $methods      = dokan_withdraw_get_active_methods();
         $currentuser  = dokan_get_current_user_id();
         $profile_info = dokan_get_store_info( dokan_get_current_user_id() );
+        $method_key   = str_replace( '/manage-', '', $slug_suffix );
+        $is_edit_mode = false;
+
+        /**
+         * This filter is only used if the Payment method has different key than the key used in meta value of
+         * meta key 'dokan_profile_settings' for that Payment method
+         *
+         * @since DOKAN_LITE_SINCE
+         */
+        $mis_match_map = apply_filters( 'payment_method_key_for_store_payment_settings', [] );
+
+        $unused_methods       = dokan_get_unused_payment_methods( $methods, $profile_info['payment'], array_values( $mis_match_map ) );
+        $unused_methods_assoc = array_reduce(
```

Comment:
Please work on your variable naming. They are not meaningful enough. Also, use comments on every step to describe what you intent to do.